bower.json

```diff
@@ -6,11 +6,11 @@
   "readmeFilename"  : "README.md",
   "main"            : "message-center.js",
   "dependencies"    : {
-    "angular"       : ">=1.2.0 <1.5.0",
-    "angular-route" : ">=1.2.0 <1.5.0"
+    "angular"       : ">=1.2.0 <1.6.0",
```

Comment:
Only if you actually run tests with this: https://github.com/angular/angular.js/releases/tag/v1.6.0-rc.0

Otherwise it's unsafe.